docs/secure-connections/sauce-connect/setup-configuration/environment-variables.md

```diff
@@ -8,21 +8,133 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
+
+
+## System proxy environment variables
+
+See the [proxy auto-configuration](/secure-connections/sauce-connect/setup-configuration/additional-proxies/#proxy-auto-configuration-automatic) documentation.
+
+| Environment Variable                                                                              | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              | Platforms           |
+|---------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------|
+| `http_proxy`<br/>`https_proxy`<br/>`HTTP_PROXY`<br/>`HTTPS_PROXY`<br/>`all_proxy`<br/>`ALL_PROXY` | Sets an HTTP proxy to be used by Sauce Connect Proxy. It can be formatted as `http://hostname:port` or `hostname:port`.<br/><br/>On Linux and Mac environments, `http_proxy` and `https_proxy` variables can contain proxy credentials in the following format: `scheme://user:password@host:port`                                                                                                                                                                                                                       | Windows, Linux, Mac |
+| `no_proxy`<br/>`NO_PROXY`                                                                         | Sets hostnames that will not be proxied, even when Sauce Connect Proxy is configured to use a proxy. Format as a comma-separated list. Subdomain wildcarding is supported when the hostname starts with a `.` Examples:<br/><br/>`saucelabs.com,spam.net`: Only requests hitting `saucelabs.com` and `spam.net` will not be proxied. All other requests will be proxied.<br/><br/>`.example.com`: All requests going to any subdomain of `example.com` will not be proxied (e.g., `one.example.com`, `asdf.example.com`) | Linux               |
+
+
+We don't recommend configuring proxy via these enviroment variables since these variables behave differently on different platforms.
```

Comment:
```suggestion
We don't recommend configuring Sauce Connect Proxy via these enviroment variables since these variables behave differently on different platforms.
```